README.md

```diff
@@ -431,6 +431,7 @@ Here is a list of Local FS settings (under `fs.` prefix)`:
 | `fs.excludes`                    | `null`        | [Includes and Excludes](#includes-and-excludes)                                   |
 | `fs.json_support`                | `false`       | [Indexing JSon docs](#indexing-json-docs)                                         |
 | `fs.xml_support`                 | `false`       | [Indexing XML docs](#indexing-xml-docs) (from 2.2)                                |
+| `fs.use_deprecated_json_setup`   | `false`       | [Use Deprecated JSon setup](#use-deprecated-json-setup)                                 |
```

Comment:
Thanks for writing this doc. I'm very sorry to say that at the end of the day, I'd prefer a setting named `add_as_inner_object` which is `false` by default (so default existing behavior).

If `true`, then we send the inner content to `object` field (instead of `json_content`).